src/kirby/scss/_native-variables.scss

```diff
@@ -1,8 +1,75 @@
- 
+
+// Always remember to keep this file and scss-helper.ts in sync
+
+
 // Import the theme’s variables. If you’re using a color scheme
 // other than “light”, switch the path to the alternative scheme,
 // for example '~nativescript-theme-core/scss/dark'.
 @import '~nativescript-theme-core/scss/light';
 
 // Place any custom variables here. For example:
-// $page-content-placeholder-font-size: 20;
\ No newline at end of file
+// $page-content-placeholder-font-size: 20;
+
+/* Page layout
+--------------------------------------------------------------------------------*/
+$page-max-width: 1366;
+
+/* Breakpoints
+--------------------------------------------------------------------------------*/
+$breakpoints: (
+  small: 361,
+  medium: 721,
+  large: 1025
+);
+
```

Comment:
This, and probably more, should be in a shared variables file for web also. :)

Reply:
Yes, den tager jeg med Tommy